hazelcast/lifecycle.py

```diff
@@ -58,7 +58,7 @@ def fire_lifecycle_event(self, new_state):
             self.is_live = False
 
         self.state = new_state
-        self.logger.debug("New Lifecycle state is %s", new_state)
+        self.logger.info("New Lifecycle state is %s", new_state)
```

Comment:
Can you also update the following outputs according to this new change?

- https://github.com/hazelcast/hazelcast-python-client#15-basic-usage
- https://github.com/hazelcast/hazelcast-python-client#791-logging-configuration
- https://github.com/hazelcast/hazelcast-python-client#setting-custom-handlers